lint/testdata/floatCompare/positive_tests.go

```diff
@@ -50,4 +50,7 @@ func f4(a, b float64) {
 	/// change `a != b` to `math.Abs(a - b) >= eps`
 	_ = a == b && b != a || !(a != b)
 
+	// TODO: change `a == b+a` to `math.Abs(a - b - a) < eps`
```

Comment:
Shouldn't it be `math.Abs(a - (b+a)) < eps`?

Reply:
Done